Assets/Scripts/Collectible.cs

```diff
@@ -18,14 +22,19 @@ public class Collectible : MonoBehaviour
     private float bounceYAmount=0.005f;
     [SerializeField]
     private Text collectibleText;
+    [SerializeField]
+    private float alphaMinus = -.001f;
 
-    private int bounceAmount, UITime=0;
-    private bool falling=true;
+    private float bounceAmount;
+    private float UITime=0;
+    private bool falling = true;
     private AudioSource audioSource;
     private SpriteRenderer spriteRenderer;
     private CircleCollider2D circleCollider;
     private bool textIsActive, textWasActive;
     private Color fillerColor;
+    const string necklace = "Necklace", gem = "Gem", ring = "WeddingRing", present = "Present";
```

Comment:
there's a C# trick for this, although I don't think it works in Unity by default until 2018.3 so don't worry about changing it for now.  https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/nameof

Reply:
e.g. `const string necklace = nameof(necklace);` then there are no human typed strings at all so no possibility of typos